components/alert/alert.css

```diff
@@ -0,0 +1,112 @@
+@import url('/css/style.css');
+
+body {
+  background-image: none;
+}
+.wrapper .bottom-nav {
+  width: 100%;
+  height: 45px;
```

Comment:
Try to use rems instead of px, 
if you want to use any specific size then you can use online px to rem converter.